internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -1230,3 +1251,17 @@ func makeIrTriggerSet(in []egv1a1.TriggerEnum) []ir.TriggerEnum {
 	}
 	return irTriggers
 }
+
+// isPrimeNumber judges whether the number is a prime number
+func isPrimeNumber(n uint64) bool {
```

Comment:
can we use the prime number check method in stdlib https://pkg.go.dev/math/big#Int.ProbablyPrime?

this method is 100% accurate for inputs less than 2⁶⁴ since our maximum table size is 5000011.

Reply:
Has replaced with function from `go/math`.